tools/cli/commands/release.js

```diff
@@ -150,12 +156,39 @@ export async function scriptRouter( argv ) {
 				  - You will now likely want to start a new release cycle like so:
 				      jetpack release ${ argv.project } new-cycle \n`.replace( /^\t+/gm, '' );
 			break;
-		case 'append':
-		case 'new-cycle':
+		case 'amend':
+			await checkBranchValid( argv );
+			argv.script = `vendor/bin/changelogger`;
+			argv.scriptArgs = [ `write`, `--amend` ];
+			argv.workingDir = `projects/${ argv.project }`;
+			argv.next = `Finished! Next:  
+				  	- You will now likely want to update readme.txt again, then commit to the release branch:
+				    	jetpack release ${ argv.project } readme \n`.replace( /^\t+/gm, '' );
```

Comment:
You've got something weird going on with the spaces and tabs here.
```suggestion
			argv.next = `Finished! Next:  
				  - You will now likely want to update readme.txt again, then commit to the release branch:
				    jetpack release ${ argv.project } readme \n`.replace( /^\t+/gm, '' );
```